xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd

```diff
@@ -210,36 +214,98 @@ Rail transport, Roads and Road transport
+				</xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="rail"/>
-			<xsd:enumeration value="intercityRail"/>
-			<xsd:enumeration value="urbanRail"/>
-			<xsd:enumeration value="metro"/>
-			<xsd:enumeration value="air"/>
-			<xsd:enumeration value="water"/>
-			<xsd:enumeration value="cableway"/>
-			<xsd:enumeration value="funicular"/>
-			<xsd:enumeration value="snowAndIce"/>
+			<xsd:enumeration value="intercityRail">
+				<xsd:annotation>
+					<xsd:documentation>Deprecated. Use Mode rail with RailSubmode longDistance.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="urbanRail">
+				<xsd:annotation>
+					<xsd:documentation>Deprecated. Use Mode metro with MetroSubmode urbanRail (or in some few cases rail with RailSubmode local). </xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="metro">
+				<xsd:annotation>
+					<xsd:documentation>Within an urban area. For underground and railway.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="air">
+				<xsd:annotation>
+					<xsd:documentation>All air related Mode. No special distinction is done.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="water">
+				<xsd:annotation>
+					<xsd:documentation>Most water related modes. The only specialisation is the Mode ferry. In Transmodel also ship was used for this mode.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="cableway">
+				<xsd:annotation>
+					<xsd:documentation>Can be only two cabines or multiple.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="funicular">
+				<xsd:annotation>
+					<xsd:documentation>cable railway on steep slope using two counterbalanced carriages.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="snowAndIce">
+				<xsd:annotation>
+					<xsd:documentation>A collection for all Mode involving snow and ice.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="taxi"/>
-			<xsd:enumeration value="ferry"/>
-			<xsd:enumeration value="lift"/>
+			<xsd:enumeration value="ferry">
+				<xsd:annotation>
+					<xsd:documentation>Ferry can be passenger ferries and/or car ferries. The specialisation from water is the detailed schedule and usually the very rigid pattern with only a few stops on the route.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="lift">
+				<xsd:annotation>
+					<xsd:documentation>General Mode for Modes moved by cable. Especially lifts are not only vertical elevators. If a better specialisation applies (like cableway) use that.</xsd:documentation>
```

Comment:
I guess this should also include all sorts of elevators? There exist hydraulic and cogwheel-driven elevators. Proposal: "Mode for elevators and for modes moved by cable...."

Reply:
Sure, make it a suggestion.

Reply:
I would rather go to the Lift Equipment description to go to that level of details (modes are expected to be generic)

Reply:
@trurlurl  can you accept Christophe's comment and we don't change it here?

Reply:
@trurlurl sorry is merged already. so done.